src/chapters/booleans-and-conditionals/nested-conditionals.rst

```diff
@@ -0,0 +1,68 @@
+Nested Conditionals
+===================
+
+We can write code with more complex branching behavior by combining conditionals, and in particular by nesting conditionals. Let's see how this works by tackling the following problem.
+
+.. admonition:: Example
+
+    Write code that prints different messages based on the value of a number variable. If the number is odd, print nothing. If it is even, print "EVEN!" If it is also positive print "POSITIVE!".
+
+Our first attempt at a solution might look like this:
+
+.. sourcecode:: js
+
+   let num = 7;
+
+   if (num % 2 === 0) {
+      console.log("EVEN!");
+   }
+
+   if (num > 0) {
+      console.log("POSITIVE!");
+   }
+
+**Output**
+
+::
+
+   POSITIVE!
+
+We find that the output is ``POSITIVE!``, even though 7 is odd and so nothing should be printed. This code doesn't work as desired because we only want to test for positivity when we already know that the number is even. We can enable this behavior by putting the second conditional *inside* the first.
+
+.. sourcecode:: js
+
+   let num = 7;
+
+   if (num % 2 === 0) {
+       console.log("EVEN!");
+
+       if (num > 0) {
+           console.log("POSITIVE!");
+       }
+   }
+
+Try this with a bunch of values (even, odd, positive, negative) to ensure it works as desired. Nice, huh?
+
```

Comment:
The words "Try this" beg for a repl link.  Playing with ``if`` statements is engaging and satisfying, especially since this is first chance the students have to really deviate from a proscribed path. ("What if I do THIS?  Huh.  That's cool.")

Reply:
Done!